common/templates/ellipses.html

```diff
@@ -21,7 +21,7 @@
         <label />
     </div>
 </td>
-<td ng-if="config.showNull || tuple.uniqueId" ng-repeat="val in rowValues track by $index">
+<td ng-if="config.showNull || tuple.uniqueId" ng-repeat="val in rowValues track by $index" ng-class="{'disabled-cell': columnModels[$index].columnError}">
```

Comment:
This is why the ellipses directive (our row directive) has all the columnModels attached to it.